code/src/App.js

```diff
@@ -1,9 +1,13 @@
-import React from 'react'
+import React from "react";
+import Form from "./components/Form";
+import "./app.css";
 
 export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
+    <div className="container">
+      <div className="main-container">
+        <Form />
+      </div>
     </div>
-  )
-}
+  );
+};
```

Comment:
I really like that you really made use of components and created a lot of them in this project which makes the code really readable.